gtsfm/utils/io.py

```diff
@@ -116,6 +118,42 @@ def read_bal(file_path: str) -> GtsfmData:
     return gtsfm_data
 
 
+def read_cameras_txt(fpath: str) -> Optional[List[Cal3Bundler]]:
+    """Read camera calibrations from a COLMAP-formatted cameras.txt file.
```

Comment:
Can we have unit tests for these methods?

Reply:
just added unit tests for them